embiggen/sequences/node2vec_sequence.py

```diff
@@ -44,16 +44,15 @@ def __getitem__(self, idx: int) -> Tuple[Tuple[np.ndarray, np.ndarray], None]:
         Tuple of tuples with input data.
         """
         return self._graph.node2vec(
-            idx,
             self._batch_size,
             self._walk_length,
             iterations=self._iterations,
             window_size=self._window_size,
-            shuffle=self._shuffle,
             min_length=self._min_length,
             return_weight=self._return_weight,
             explore_weight=self._explore_weight,
             change_node_type_weight=self._change_node_type_weight,
             change_edge_type_weight=self._change_edge_type_weight,
-            dense_nodes_mapping=self._dense_nodes_mapping
```

Comment:
spelling -- should be ``dense_node_mapping``

Reply:
Fixed: since this is also a typo within ensmallen, it will break the current branch. It will be fixed in the next version of ensmallen that we should release this afternoon.